_config.yml

```diff
@@ -8,16 +8,19 @@
 
 title: fastpages
 description: An easy to use blogging platform for data scientists, with support for Jupyter Notebooks.
-github_username: jph00
+github_username: fastai
```

Comment:
Sure you want fastai as a default? We might get a few mentions of other people blogs that way, probably wouldn't hurt too much ;)

Reply:
This is so the colab/github badges in our demo have the right link because the links are built with logic:

`{{site.github_username}}/{{site.github_repo...`